lib/active_remote/attribute_methods.rb

```diff
@@ -12,8 +12,11 @@ def [](name)
       attribute(name)
     end
 
-    def []=(name, value)
-      write_attribute(name, value)
+    def []=(attr_name, value)
+      name = attr_name.to_s
+      name = self.class.attribute_aliases[name] || name
+
+      @attributes.write_from_user(name, value)
```

Comment:
I like what you did here. There is still a public `write_attribute` method for active_record (https://apidock.com/rails/v6.1.3.1/ActiveRecord/AttributeMethods/Write/write_attribute), but I think what you added here makes sense.

Reply:
I think it was ActiveModel that removed the method, and that's what we were using previously.

Reply:
I think this is wrong. I think I ended up copying an implementation that was an overridden implementation.